source/util.js

```diff
@@ -61,3 +64,18 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.checkIgnoreStrategy = ({files}) => {
```

Comment:
I think that this method should be moved to `source/npm/util.js` since it's specific to npm.

Reply:
Moved.